qiskit_aqua/algorithms/adaptive/qsvm/qsvm_variational.py

```diff
@@ -226,6 +236,31 @@ def _get_prediction(self, data, theta):
 
         return predicted_probs, predicted_labels
 
+    # Breaks data into minibatches. Labels are optional, but will be broken into batches if included.
+    def batch_data(self, data, labels=None):
+        label_batches = None
+
+        if self._minibatch_size > 0 and self._minibatch_size < len(data):
+            batch_size = min(self._minibatch_size, len(data))
+            if labels is not None:
+                shuffled_samples, shuffled_labels = shuffle(data, labels)
```

Comment:
could we reuse the `random_state` in the QuantumAlgorithm to control the randomness?
`suffle(..., random_state=self.random)`